packages/p2p/src/stores/my-ads-store.js

```diff
@@ -72,40 +72,41 @@ export default class MyAdsStore extends BaseStore {
     @action.bound
     getAdvertInfo() {
         this.setIsFormLoading(true);
-
         requestWS({
             p2p_advert_info: 1,
             id: this.selected_ad_id,
-        }).then(response => {
-            if (!response.error) {
-                const { p2p_advert_info } = response;
-                if (!p2p_advert_info.payment_method_names)
-                    p2p_advert_info.payment_method_names = this.payment_method_names;
-                if (!p2p_advert_info.payment_method_details)
-                    p2p_advert_info.payment_method_details = this.payment_method_details;
-                this.setP2pAdvertInformation(p2p_advert_info);
-            }
-            this.setIsFormLoading(false);
-        });
+        })
+            .then(response => {
+                if (response && !response.error) {
```

Comment:
Why aren't we handling errors here? Can you add it? Or at least add a TODO for it.